piton/design/chipset/rtl/storage_addr_trans.v.pyv

```diff
@@ -33,7 +33,7 @@ import pyhplib
 import os
 from pyhplib import *
 
-if PITON_ARIANE:
+if PITON_BLACKPARROT:
```

Comment:
Didn't we add `PITON_RV64_PLATFORM`? Is that available in pyhplib.py? This applies here and in `storage_addr_trans_unified.v.pyv`